packages/create-block/lib/init-package-json.js

```diff
@@ -24,6 +24,7 @@ module.exports = async ( {
 	npmDependencies,
 	npmDevDependencies,
 	customScripts,
+	isDynamic,
```

Comment:
Here and in other places in the file:
```suggestion
	isDynamicVariant,
```